site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -348,6 +348,37 @@ Workloads as possible, preferring Workloads with these characteristics:
 - Workloads with the lowest priority.
 - Workloads that have been admitted more recently.
 
+## FlavorFungibility
+
+When there is not enough nominal quota of resources in a ResourceFlavor, the incoming workload can borrow or preempt in ClusterQueue or Cohort. 
+
+By default the incoming workload will stop trying the next flavor if it can get enough resource by borrowing. 
+And preemption will be triggered after all ResourceFlavors were considered.
+
+ClusterQueues can also try preemption whenever preemption might help before trying next ResourceFlavor or only borrowing after all ResourceFlavors have been considered by setting `FlavorFungibility`.
+
+A configuration for a ClusterQueue that changes this behavior looks like the following:
+```
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: "team-a-cq"
+spec:
+  flavorFungibility:
+    whenCanBorrow: TryNextFlavor
+    whenCanPreempt: Preempt
+```
+
+The fields above have the following semantics:
```

Comment:
```suggestion
The fields above do the following:
```

Reply:
https://github.com/kubernetes-sigs/kueue/blob/40bd411beddb5bd415d880a0c841c92ababc0458/site/content/en/docs/concepts/cluster_queue.md?plain=1#L324C47-L324C47

I have also made modifications to the identical expression within the text.